src/config/createFilesConfig.json

```diff
@@ -0,0 +1,10 @@
+[
+	[ "package.json", "." ],
+	[ "typings.json", "." ],
+	[ "tsconfig.json", "." ],
+	[ "tslint.json", "." ],
```

Comment:
What do we think then? Should `tslint` and `editorconfig` be there? I'll raise an issue for discussion